test/src/module/config/parseTest.c

```diff
@@ -1335,6 +1335,7 @@ testRun(void)
         TEST_RESULT_STR_Z(cfgOptionIdxStr(cfgOptPgSocketPath, 0), "/path/to/socket", "    pg1-socket-path is set");
         TEST_RESULT_INT(cfgOptionIdxSource(cfgOptPgSocketPath, 0), cfgSourceConfig, "    pg1-socket-path is config param");
         TEST_RESULT_BOOL(cfgOptionBool(cfgOptOnline), false, "    online not is set");
+        TEST_RESULT_STR_Z(cfgOptionDisplay(cfgOptOnline), "false", "    online display is false");
```

Comment:
I feel like we are trying to do a bit too much in this PR and not actually accomplishing the requirement: "If the value was set by the user via the command line, config, etc., then that exact value will be displayed." 
"true" and "false" are never used as values - only "y" and "n". The timeouts are an issue and even there they are inconsistent (see previous comment). I think we should rethink the requirements and what is trying to be accomplished. We might just want to solve the timeout/size problem as that appears to be what motivated this PR. Having a general Display function that is only useful sometimes may not be the answer.

Reply:
I did this because we have messages that use true/false the output booleans, e.g. backup.c lines 434 and 455. I didn't think this commit should try to go through and update existing messages to y/n, though I agree this makes more sense.

Agreed that this is all a bit messy but the goal here is to centralize some logic, not make all the error messages consistent.

Reply:
I don't feel it is simplifying the option display - I think it is now just adding more ways to display it, which can get confusing on which to use.

Reply:
We definitely need to make boolean messages more consistent, but as discussed I have created a card for it: https://github.com/pgbackrest/pgbackrest/projects/2#card-59948715